### DISCUSSION TITLE
Best solution for multiple MVC websites using the same database

### DISCUSSION THREAD

**Discussion by tombal98** [OrchardCMS/OrchardCore#8167] [Q&A]
Hi, i'm looking for the best solution using Orchard Core, to make 4 websites sharing some contents, some content types and having also for each website it's own content and theme. What is the best way to do that : using tenants or headless or as a decoupled CMS. 

I want only one database for all the websites. Each site will have it's own contents using same content types for all, using taxonomy to select the contents i'll use on a site. 

Can i do that with Decoupled CMS, using Orchard to manage all content types, contents, security and a MVC project for each websites, using content services to access the contents needed for each one. 

I'm little confused between using Decoupled CMS and tenants for this kind of project.  

Hoping to have been clear in my request

Thanks for your answers

**Comment by Huyvictory** [comment]
Have you find the suitable solution ?
From your description, i think the headless CMS Orchard will be the one that you should consider. Since it provides GraphQL to be consumed by your client applications.
